integreat_cms/api/v3/chat/utils/chat_bot.py

```diff
@@ -130,6 +130,7 @@ def process_user_message(
             internal=True,
             automatic_message=True,
         )
+    zammad_chat.processing_answer = False
```

Comment:
I just realized - the `zammad_chat` never gets saved, right?

Reply:
Ah yes, this line was committed accidentally. Initially I wanted to store this information in the model but then went on to just store it in the cache.

Reply:
Alright, makes sense. What threw me was just that this is an object attribute which never gets saved. IMO a short comment explaining that this is intentional would be great.